deploy/demo-cli/README.md

```diff
@@ -9,15 +9,19 @@ $ ./blox-list-environments.py --help
 == Blox Demo CLI - List Blox Environments ==
 
 usage: blox-list-environments.py [-h] [--region REGION] [--apigateway]
-                                 [--host HOST]
+                                 [--host HOST] [--environment ENVIRONMENT]
```

Comment:
Not sure I understand. ListEnvironments doesn't take environment as a parameter does it?